src/OrbitGl/App.cpp

```diff
@@ -266,10 +269,28 @@ OrbitApp::~OrbitApp() {
 }
 
 void OrbitApp::OnCaptureFinished(const CaptureFinished& capture_finished) {
-  if (capture_finished.status() == CaptureFinished::kFailed) {
-    SendErrorToUi("Capture Failed", capture_finished.error_message());
-    ERROR("Capture Finished with error: %s", capture_finished.error_message());
-  }
+  main_thread_executor_->Schedule([this, capture_finished]() {
+    switch (capture_finished.status()) {
+      case orbit_grpc_protos::CaptureFinished::kSuccessful: {
+        main_window_->AppendToCaptureLog(MainWindowInterface::CaptureLogSeverity::kInfo,
+                                         GetCaptureTime(), "Capture finished.");
+      } break;
+      case orbit_grpc_protos::CaptureFinished::kFailed: {
+        SendErrorToUi("Capture Failed", capture_finished.error_message());
+        ERROR("Capture Finished with error: %s", capture_finished.error_message());
+        main_window_->AppendToCaptureLog(
+            MainWindowInterface::CaptureLogSeverity::kError, GetCaptureTime(),
+            absl::StrFormat("Capture finished with error: %s.", capture_finished.error_message()));
+      } break;
+      case orbit_grpc_protos::
+          CaptureFinished_Status_CaptureFinished_Status_INT_MIN_SENTINEL_DO_NOT_USE_:
+        [[fallthrough]];
```

Comment:
I think in case of completely empty case we do not want to use [[fallthrough]], it actually makes it harder to read.

I would reserve it only for cases where there is some work to be done and after that fallthrought is intended.